resample-1.8.1/sndlib-20/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 
-IF(WIN32 OR CYGWIN)
+IF(WIN32 OR CYGWIN OR HAIKU)
```

Comment:
Isn't Haiku a Linux-ish OS rather than Windows? Don't know it though...

Reply:
There is no jack, alsa, oss, etc on Haiku, but i will check if this actually required.